src/components/charts/SummaryPieChart.jsx

```diff
@@ -0,0 +1,162 @@
+
+import {
+  PieChart, Pie, Tooltip, Cell,
+} from 'recharts';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const PieChartWrapper = styled.div`
+  width: 100%;
+  padding: 12px;
+  background-color: white;
+`;
+
+const PieChartTitle = styled.div`
+  text-align: center;
+  border-bottom: solid black 1px;
+  line-height: 10px;
+  height: 22px;
+`;
+
+const PieChartInnerWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const PieChartLegendWrapper = styled.div`
+  width: 70px;
+  flexGrow: 0;
+  padding-top: 8px;
+`;
+
+const pieChartStyle = {
+  flexGrow: 1,
+  display: 'flex',
+  justifyContent: 'flex-end',
+  marginTop: '22px',
+};
+
+const LegendWrapper = styled.div`
```

Comment:
Do we need this since we already have `PieCharLegendWrapper`?

Reply:
`LegendWrapper` is child items inside `PieChartLegendWrapper` so we still need it to set margin of each child. Sorry I am bad at creating names